registry/registry.go

```diff
@@ -219,6 +220,10 @@ func ContinueOnError(err error) bool {
 		return shouldV2Fallback(v)
 	case *client.UnexpectedHTTPResponseError:
 		return true
+	case error:
+		if val := strings.Contains(err.Error(), strings.ToLower(syscall.ENOSPC.Error())); val {
```

Comment:
This is an error decoded from a reexec stdout, I odn't think you can compare the error here... but I could be wrong.